app/_src/guides/consumer-producer-policies.md

```diff
@@ -15,30 +15,13 @@ This guide will help you get comfortable with producer consumer model.
 ## Basic setup
 
 In order to be able to fully utilize namespace scoped policies you need to use [MeshService](/docs/{{ page.version }}/networking/meshservice). 
```

Comment:
Should we just remove this line and just consider the user as using `MeshService` ?

Reply:
I was thinking about this, but I don't think we can assume users are using MeshService, this guide is linked in blog post for release, and there can be users coming to check out new feature with already existing environments. Without starting with new quickstart